packages/py3_sphinxcontrib_serializinghtml.rb

```diff
@@ -7,6 +7,7 @@ class Py3_sphinxcontrib_serializinghtml < Pip
   license 'BSD'
   compatibility 'all'
   source_url 'SKIP'
+  binary_compression 'tar.zst'
```

Comment:
this package still has no_compile_needed, so no point in a `binary_compression`  value